repos/backend/libs/definitions/definitions.js

```diff
@@ -24,9 +24,15 @@ const parseDefinitions = definitionFiles => {
 }
 
 const loadDefinitions = async config => {
-  const { stepsFolder } = config.paths
-  const definitionFiles = stepsFolder && await loadDefinitionsFiles(stepsFolder)
-  const definitions = await parseDefinitions(definitionFiles) || []
+  const { stepsDir, testsRoot } = config.paths
+  const pathToSteps = path.join(testsRoot, stepsDir)
+  const definitionFiles = stepsDir && await loadDefinitionsFiles(pathToSteps)
+  const herkinDefinitionFiles = await loadDefinitionsFiles('/keg/tap/repos/testUtils/bdd/steps')
```

Comment:
does this need to be hardcoded?

Reply:
as discussed, updated to use global constants